fastlane/lanes/test.rb

```diff
@@ -19,40 +17,15 @@
 
     gradle(tasks: ["WordPress:assemble#{app.to_s.capitalize}VanillaDebug", "WordPress:assemble#{app.to_s.capitalize}VanillaDebugAndroidTest"])
 
-    # Run the instrumented tests in Firebase Test Lab
-    firebase_login(
-      key_file: GOOGLE_FIREBASE_SECRETS_PATH
-    )
-
-    apk_dir = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'build', 'outputs', 'apk')
-
-    test_succeeded = android_firebase_test(
-      project_id: firebase_secret(name: 'project_id'),
-      key_file: GOOGLE_FIREBASE_SECRETS_PATH,
-      model: 'Pixel2.arm',
-      version: 30,
-      test_apk_path: File.join(apk_dir, 'androidTest', "#{app}Vanilla", 'debug', "org.wordpress.android-#{app}-vanilla-debug-androidTest.apk"),
-      apk_path: File.join(apk_dir, "#{app}Vanilla", 'debug', "org.wordpress.android-#{app}-vanilla-debug.apk"),
-      test_targets: 'notPackage org.wordpress.android.ui.screenshots',
-      results_output_dir: File.join(PROJECT_ROOT_FOLDER, 'build', 'instrumented-tests'),
-      crash_on_test_failure: false
-    )
-
     annotation_ctx = "firebase-test-#{app}-vanilla-debug"
-    if test_succeeded
+    begin
+      gradle(task: "runFlank#{app.to_s.capitalize}")
       sh("buildkite-agent annotation remove --context '#{annotation_ctx}' || true") if is_ci?
-    else
-      details_url = lane_context[SharedValues::FIREBASE_TEST_MORE_DETAILS_URL]
+    rescue
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-Android/blob/cd01ef0f7d5be67700456972cf7ef66609d5dea5/fastlane/lanes/test.rb#L24"></span>Style/RescueStandardError: Avoid rescuing without specifying an error class.</td>
    </tr>
  </tbody>
</table>

Reply:
@iangmaia could you please confirm if it's okay to ignore? I want to rescue from all possible causes of `gradle` or `sh` crash.

Reply:
I think those would still be caught if using `rescue StandardError`?

```suggestion
    rescue StandardError
```

By the way, mentioning just in case (not sure if it would make more sense here, as you also have the `gradle` call): for `sh`, you [can also use](https://docs.fastlane.tools/actions/sh/) a `error_callback` parameter if you want to do a custom handling in case the command returns an error.

Reply:
True, `rescue StandardError` worked fine, thanks! Addressed in 997e8087676b9c8110fc676501802c1e823edfc1

About `error_callback` - I don't see it being mentioned in `gradle` action documentation, and Gradle part is the most crucial part of this handling so I think we could keep this try/catch approach